src/main/webapp/app/pool/pool-list/pool-list.component.ts

```diff
@@ -9,12 +12,27 @@ import { Pool } from 'app/pool/pool';
 })
 export class PoolListComponent implements OnInit {
     pools: Pool[];
+    public displayName: string;
+    public minimum: number;
+    public maximum: number;
+    public enabled: string;
+    public selectedTools: Tool[] = [];
+    public tools: Observable<any[]>;
```

Comment:
Why Observable?

Reply:
Regular array doesn't work with ng-select in this case, must be observable

Reply:
Ok